src/modules/battery.cpp

```diff
@@ -30,7 +30,7 @@ namespace modules {
     m_lastpoll = chrono::system_clock::now();
 
     auto path_adapter = string_util::replace(PATH_ADAPTER, "%adapter%", m_conf.get(name(), "adapter", "ADP1"s)) + "/";
-    auto path_battery = string_util::replace(PATH_BATTERY, "%battery%", m_conf.get(name(), "battery", "BAT0"s)) + "/";
+    auto path_battery = string_util::replace(PATH_BATTERY, "%battery%", m_conf.get(name(), "battery", "BAT0"s)) + "/";	
```

Comment:
Remove trailing space to avoid it being dropped on next save if remove trailing space is used. This helps to see what actually changed in a commit. Would also recommend setting your editor to automatically do it.